test/e2e/configmap.go

```diff
@@ -181,7 +181,7 @@ var _ = Describe("ConfigMap", func() {
 		// Kubelet projects the update into the volume and the container picks
 		// it up. This timeout is based on the default Kubelet sync period (1
 		// minute) plus additional time for fudge factor.
-		const podLogTimeout = 90 * time.Second
+		const podLogTimeout = 150 * time.Second
```

Comment:
I think the problem here is that my timeout in the original test was too optimistic.  It may take a full sync interval in the kubelet to pick up the change; bumping way up to avoid future timeouts.